packages/react-admin/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
This should be updated

Reply:
I have removed all the `packages/react-admin` folder, so I think that it is no longer needed.